docs/source/_static/switcher.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "name": "23.10.x (latest)",
```

Comment:
The information which is latest or which is stable should not be hardcoded, as that's a configration thing on the RTD side

Reply:
This file doesn't do anything. If we ever decide to use the built version switcher for the pydata-sphinx-theme we will need this file, but we currently don't, so it's safe to remove it.

Reply:
I will delete it.

Reply:
Sounds good, didn't know that